codegenerator/cli/npm/envio/src/sources/HyperSyncClient.res

```diff
@@ -315,19 +315,19 @@ module ResponseTypes = {
 
   // TODO: update some of these fields to be integers rather than strings.
   type authorizationList = {
-    chainId: string,
+    chainId: bigint,
     address: Address.t,
-    nonce: string,
-    yParity: string,
+    nonce: int,
+    yParity: int, // 0 or 1
```

Comment:
If genType supports it:
```suggestion
    yParity: [#0 | #1],
```

Reply:
![image](https://github.com/user-attachments/assets/cb61d1ed-e281-4a03-bafe-c17e795da3b8)
Works!